ocfweb/lab_reservations/templates/lab_reservations/pending.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class="ocf-content-block">
+        <p>Your lab reservation request has successfully
+           been submitted and is pending approval!
```

Comment:
Indentation here is a bit strange, I'd probably do something like this instead to make it clearer:

```
<p>
    Your lab reservation request has successfully
    been submitted and is pending approval!
</p>
```